src/main/java/com/deswaef/spring/examples/datajpa/util/Constants.java

```diff
@@ -12,6 +12,7 @@
 	
 	public static final int USER_ALREADY_REGISTERD = 301;
 	public static final int FILE_DOES_NOT_EXIST = 302;
+	public static final int FILE_DELETED_SUCCESSFULLY = 204;
```

Comment:
Add Proper Code. The code should be added to the successful code and value should be 104.